### PR TITLE
Modify RIF tests so there are always DME claims.

### PR DESCRIPTION
**Pull request by jawalonoski**
Modify RIF tests so there are always DME claims.

- Adds a hearing aid module for only the unit tests.
- Adds a mock DME mapper class to ensure the hearing aid claim is exported regardless of the presence of mapping files.
- Ignore sonatype-lift complaints about "God Classes" -- we hear you, now go away.

### CODE REVIEW

**Comment by jawalonoski** [comment]
> ⚠️ **26 God Classes** were detected by Lift in this project. [Visit the Lift web console](https://lift.sonatype.com/results/github.com/synthetichealth/synthea/01GFRKG7Q6C7T1T4B6YZTMV81Y?tab=technical-debt&utm_source=github.com&utm_campaign=lift-comment&utm_content=synthetichealth%20synthea) for more details.

@sonatype-lift exclude refactor-first

File: .lift.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+disableTools = [ "refactor-first" ]
```
[the remark this answers is not in the document]
Comment (jawalonoski) [reply .lift.toml:1]: @sonatype-lift exclude refactor-first

File: src/test/java/org/mitre/synthea/export/BB2RIFExporterTest.java [hunk 2]
```diff
@@ -45,9 +47,32 @@
...
+
+    public boolean canMap(String codeToMap) {
+      return true;
+    }
+
+    public boolean hasMap() {
+      return true;
+    }
+
+    public String map(String codeToMap, String bfdCodeType, RandomNumberGenerator rand,
```
Comment (hadleynet) [inline src/test/java/org/mitre/synthea/export/BB2RIFExporterTest.java:66]: I got a checkstyleTest warning about this function missing a Javadoc comment.

**Review by hadleynet — CHANGES_REQUESTED** [review]
Nice! Just one minor check style warning to fix and then it is good to go.

File: src/test/java/org/mitre/synthea/export/BB2RIFExporterTest.java (hunk 2 above)
Comment (jawalonoski) [reply src/test/java/org/mitre/synthea/export/BB2RIFExporterTest.java:74]: Should be fixed now.

**Review by hadleynet — APPROVED** [review]

**Pull request merged**